mmv1/third_party/terraform/services/privateca/privateca_ca_utils.go

```diff
@@ -228,12 +228,14 @@ func activateSubCAWithFirstPartyIssuer(config *transport_tpg.Config, d *schema.R
 		return fmt.Errorf("Error creating Certificate: %s", err)
 	}
 	signedCACert := res["pemCertificate"]
+	signerCertChain := res["pemCertificateChain"]
 
 	// 4. activate sub CA with the signed CA cert.
 	activateObj := make(map[string]interface{})
 	activateObj["pemCaCertificate"] = signedCACert
 	activateObj["subordinateConfig"] = make(map[string]interface{})
-	activateObj["subordinateConfig"].(map[string]interface{})["certificateAuthority"] = issuer
+	activateObj["subordinateConfig"].(map[string]interface{})["pemIssuerChain"] = make(map[string]interface{})
```

Comment:
Can you explain what is happening here? Why do we change from setting the subordinateConfig.certificateAuthority to setting the subordinateConfig.pemissuerChain.pemCertificates?

Reply:
Please find details at b/332548736